src/index.ts

```diff
@@ -4,21 +4,55 @@
 import './index.css';
 
 import { IconH1, IconH2, IconH3, IconH4, IconH5, IconH6, IconHeading } from '@codexteam/icons';
+import { API, BlockTune, PasteEvent } from '@editorjs/editorjs';
 
 /**
- * @typedef {object} HeaderData
- * @description Tool's input and output data format
- * @property {string} text — Header's content
- * @property {number} level - Header's level from 1 to 6
- */
+* @description Tool's input and output data format
+*/
+export interface HeaderData {
+  /** Header's content */
+  text: string;
+  /** Header's level from 1 to 6 */
+  level: number;
+}
 
 /**
- * @typedef {object} HeaderConfig
  * @description Tool's config from Editor
- * @property {string} placeholder — Block's placeholder
- * @property {number[]} levels — Heading levels
- * @property {number} defaultLevel — default level
  */
+interface HeaderConfig {
```

Comment:
Could be also used outside, do lets add export
```suggestion
export interface HeaderConfig {
```